tests/Microsoft.Bot.Builder.AI.QnA.Tests/QnAMakerTests.cs

```diff
@@ -198,43 +195,42 @@ public async Task QnaMaker_TraceActivity()
                 await context.SendActivityAsync("echo:" + context.Activity.Text);
             })
                 .Send("how do I clean the stove?")
-                    .AssertReply((activity) => Assert.AreEqual(activity.Type, ActivityTypes.Typing))
+                    .AssertReply((activity) => Assert.Equal(activity.Type, ActivityTypes.Typing))
                     .AssertReply("echo:how do I clean the stove?")
                 .Send("bar")
-                    .AssertReply((activity) => Assert.AreEqual(activity.Type, ActivityTypes.Typing))
+                    .AssertReply((activity) => Assert.Equal(activity.Type, ActivityTypes.Typing))
                     .AssertReply("echo:bar")
                 .StartTestAsync();
 
             // Validate Trace Activity created
             var pagedResult = await transcriptStore.GetTranscriptActivitiesAsync("test", conversationId);
-            Assert.AreEqual(7, pagedResult.Items.Length);
-            Assert.AreEqual("how do I clean the stove?", pagedResult.Items[0].AsMessageActivity().Text);
-            Assert.IsTrue(pagedResult.Items[1].Type.CompareTo(ActivityTypes.Trace) == 0);
+            Assert.Equal(7, pagedResult.Items.Length);
+            Assert.Equal("how do I clean the stove?", pagedResult.Items[0].AsMessageActivity().Text);
+            Assert.True(pagedResult.Items[1].Type.CompareTo(ActivityTypes.Trace) == 0);
```

Comment:
Maybe we could change it to something like this.
```suggestion
            Assert.Equal(0, pagedResult.Items[1].Type.CompareTo(ActivityTypes.Trace));
```

Reply:
Done. Thanks!